cartography/intel/aws/ec2.py

```diff
@@ -111,6 +152,22 @@ def load_ec2_instances(session, data, region, current_aws_account_id, aws_update
     SET r.lastupdated = {aws_update_tag}
     """
 
+    ingest_key_pair = """
+    MERGE (keypair:KeyPair:EC2KeyPair{arn: {KeyPairARN}})
+    ON CREATE SET keypair.firstseen = timestamp()
+    SET keypair.keyname = {KeyName}, keypair.region = {Region}, keypair.lastupdated = {aws_update_tag}
+    WITH keypair
+    MATCH (aa:AWSAccount{id: {AWS_ACCOUNT_ID}})
+    MERGE (aa)-[r:RESOURCE]->(keypair)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    with keypair
+    MATCH (instance:EC2Instance{instanceid: {InstanceId}})
+    MERGE (instance)<-[r:LOGIN_KEY]-(keypair)
```

Comment:
Rename `LOGIN_KEY` to something more specific.  I'm just brainstorming, but maybe something like `EC2_LOGIN_KEY`?  

Or... since having the associated private key means that you are authorized to log in to the instance, maybe something like `AUTHZ_TO`?  

Any other ideas for names @ecdavis?

Reply:
How about `SSH_LOGIN_TO`?